de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/communication/extensions/UserListExtension.java

```diff
@@ -34,19 +34,22 @@ public void addUser(User user, long flags) {
 
         public long flags;
         public JID jid;
+        public boolean isServer;
         public int colorID;
         public int favoriteColorID;
         public Permission permission;
 
         private static UserListEntry create(User user, long flags) {
-            return new UserListEntry(user.getJID(), user.getColorID(),
-                user.getFavoriteColorID(), user.getPermission(), flags);
+            return new UserListEntry(user.getJID(), user.isServer(),
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "getJID"; it is deprecated. ([squid:CallToDeprecatedMethod](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3ACallToDeprecatedMethod))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=284&issues=AWarMreBq3jbRxE0bMup&open=AWarMreBq3jbRxE0bMup)

[ISSUE_KEY]: AWarMreBq3jbRxE0bMup